### DISCUSSION TITLE
What is the definition of "Premium" support?

### DISCUSSION THREAD

**Discussion by acwalter** [opengisch/QField#4250] [Q&A]
I have a paid organization account and need support, but my support ticket (through the ticket system) has not been answered or even acknowledged in 6 days. The product comparison differentiates community support from Organization accounts which have "Premium" support, but I don't see a definition of what this means. 
![image](https://user-images.githubusercontent.com/3827805/235533030-f28b6956-9ff4-45d2-921b-8071ca3d1e09.png)

**Comment by fiermanb** [comment]
Same thing here. We pay for a premium subscription, sent in a couple of emails which never got answered too.
